src/main/java/mate/controller/IndexController.java

```diff
@@ -1,9 +1,37 @@
 package mate.controller;
 
+import java.io.IOException;
+import java.time.LocalDateTime;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 import mate.dao.MyCoolResource;
 
+@WebServlet(urlPatterns = "/info")
 public class IndexController extends HttpServlet {
     private MyCoolResource myResource;
     //TODO: implement
+    
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        myResource.write(LocalDateTime.now().toString());
+        req.getRequestDispatcher("/WEB-INF/views/info.jsp").forward(req, resp);
```

Comment:
If I am not mistaken, today on Q&A it was mentioned that for the  naming consistency name of our custom  servlet should correspond with the name of jsp file it is related to (index instead of info)